docs/sampling/README.md

```diff
@@ -0,0 +1,242 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Sampling
+--->
+
+# Semantic Conventions for Sampling
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+<!-- tocstop -->
+
+These attributes reflect the effect of sampling in a telemetry
+collection pipeline.  These attributes describe how items of telemetry
+were collected, making it possible to define Span-to-Metrics pipelines
+and Logs-to-Metrics pipelines, which accurately count Spans and Log
+Records of telemetry, before sampling, in a probabilistic sense.
+
+These attributes MAY be modified by components in a collection
+pipeline to convey successive sampling that has been carried out for a
+particular item of telemetry, using the conventions for consistent
+sampling described here.  In that sense, producers and consumers
+should see these attributes as telemetry metadata.
+
+The attributes described here use non-standard naming conventions.
```

Comment:
thank you for the explanation, I think we don't need to add it to the doc itself. Also I'd still suggest removing references to OpenTracing - they provide great historical context, but don't seem practical

Reply:
Removed it, thanks!